README.md

```diff
@@ -194,7 +194,11 @@ const repo = <IPFS Repo instance or repo path>
 const node = new IPFS({
   repo: repo,
   init: true, // default
-  // init: false,
+  // init: false, // If you have already initialized the repo via `new IPFS()`.
+  //              // Be careful using this on an IPFSRepo instance you have
+  //              // initialized yourself -- the IPFS constructor initializes
+  //              // repos with extra options, so calling `init()` on an
+  //              // IPFSRepo may not always be compatible with IPFS.
```

Comment:
I'm not sure I don't understand what this comment is trying to prevent the user to do. Either a user knows what they are doing and they know supposed pitfall or they don't and the terms here don't give them any other help.

Reply:
OK. We clearly have users who don’t know what this is for as noted in the summary — this is a result of #1245.

What language would you suggest better clarifies when to and not to use `init: false`? What should I say here that helps someone like that know they can’t just:

```js
const repo1 = new IPFSRepo('/tmp/ipfs-repo1');
repo1.init({}, (err) => {
  if (err) { console.log(err); return; }

  const node1 = new IPFS({
    repo: repo1,
    init: false,
    start: true
  });

  node1.on('start', () => {
    console.log('IPFS node 1 started.');
  });
})
```

Something like:

> Use this if the repo has previously been initialized by passing it to `new IPFS()`. You should not attempt to call `init()` yourself on an IPFSRepo instance and then set `init: false` here.

?

Clearly this could go into a lot more detail, but because this is all inline in a code sample, there’s not a whole lot of room.

Reply:
I think part of the confusion with repo is how you right code so that it works both the first time its called, and following times.  init confused me a lot when I first saw it. Do I 
a) set init=true - in which case I would assume it will clear the repo every time, losing all saved data or 
b) set init=false in which case it will fail the first time. 

I think most distributed IPFS based apps are going to want to default to "initialize a new repo if it doesn't exist, or if its incompatible (because IPFS has changed version and broken old repo's) and otherwise leave it alone".  I still have no clue wither that is to set it to false, true or leave it missing (which is what we currently do. ) 

Some comments here about when to set it to true, when to false, and when to omit it would be useful.